package.json

```diff
@@ -220,7 +220,7 @@
     "dotenv": "^16.4.5",
     "dotenv-webpack": "^8.1.0",
     "electron": "^9.4.4",
-    "electron-builder": "^22.14.10",
+    "electron-builder": "23.6.0",
```

Comment:
potentially dangerous but equally helpful. the main reason for updating this is to fix issues related to running on newer machines. see 119ebdc3518bf50b561fb9aad65bc7334f56e7ce for more details.

read through the changelogs between our previous version and this version and nothing stood out to me as a major issue waiting to happen. however, open to addressing the original issue differently if we're worried about that

Reply:
Seems to work so I'm ok with this.